linux_os/guide/system/accounts/accounts-pam/locking_out_password_attempts/accounts_passwords_pam_tally2_deny_root/ansible/shared.yml

```diff
@@ -4,6 +4,11 @@
 # complexity = low
 # disruption = low
 
+{{% if product in ["sle12","sle15"] %}}
```

Comment:
The rule is restricted to sle products. So, why the conditionals here? Would be simpler to only update the file.

Reply:
Hello @marcusburghardt 
Thank you for your feedback.
If you check tests , ansible and bash - you will see that ubuntu is also use it. Because of ubuntu I added these if / else statements.  
Have a nice day
Rumen